### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by koda-11** [AILab-CVC/SEED#22]
Do you have a plan to release training code?

**Comment by geyuying** [comment]
We have released the training code of SEED-LLaMa, including SEED tokenizer, Multimodal LLM pretraining and instruction tuning. Our Multimodal LLM training codebase supports 1. large-scale multi-node training with deepspeed 2. highly-efficient multiple training datapipes.
